_data/accessibility-tests/button-group.yml

```diff
@@ -0,0 +1,85 @@
+title: Button group
+component_status: pass
+test_items:
+# General tests
+  - summary: Color is not the only method used to convey button meaning.
+    summary_additional: When you view a buttons in a group, you can see and read text, and do not have to rely on color alone for meaning.
```

Comment:
Should be "When you view buttons in a group..." (remove the "a")

Reply:
Updated in [7ffca7d](https://github.com/uswds/uswds-site/pull/3009/commits/7ffca7d5140778a2c653304845d5082e45ebe2e7)